index.bs

```diff
@@ -2431,8 +2431,8 @@ When this method is invoked, the [=authenticator=] MUST perform the following pr
     If the user does not [=user consent|consent=], return an error code equivalent to
     "{{NotAllowedError}}" and terminate the operation.
 
-1. Let |processedExtensions| be the result of [=authenticator extension processing=] for each supported [=extension
-    identifier=]/input pair in |extensions|.
+1. Let |processedExtensions| be the result of [=authenticator extension processing=] [=map/for each=] supported <var
+    ignore>extensionId</var> → <var ignore>authenticatorExtensionInput</var> in |extensions|.
```

Comment:
Why did you remove the indexed term [=extension identifier=]?  If anything, I think this should say "... supported [=extension identifier=] -> [=authenticator extension input=] pair".  Please keep the indexed term and add the second one.

Also, why were you using "var ignore"?  I don't understand your goals here.

Reply:
I made this change to closer reflect the usage description of `map/for each` in Infra. The `ignore` on the `<var>`s was then necessary to make Bikeshed not complain about that the variables were only used once.

I'll change that back to using the indexed terms.